collector/mongod/metrics.go

```diff
@@ -397,7 +397,7 @@ type CursorStats struct {
 // Export exports the cursor stats.
 func (cursorStats *CursorStats) Export(ch chan<- prometheus.Metric) {
 	metricsCursorTimedOutTotal.Set(cursorStats.TimedOut)
-	metricsCursorOpen.WithLabelValues("noTimeout").Set(cursorStats.Open.NoTimeout)
+	metricsCursorOpen.WithLabelValues("timed_out").Set(cursorStats.Open.NoTimeout)
```

Comment:
Very interesting, no we are not :)

"metrics.cursor.open.noTimeout
New in version 2.6.

The number of open cursors with the option DBQuery.Option.noTimeout set to prevent timeout after a period of inactivity."

In this case I think the original "noTimeout" is more appropriate, to represent the noTimeout driver option.